coordinator/migrations/2023-07-27-020530_intercept_id_channels/up.sql

```diff
@@ -0,0 +1,5 @@
+-- Your SQL goes here
+ALTER TABLE
+    channels
+ADD
+    COLUMN "fake_scid" TEXT UNIQUE;
```

Comment:
🔧 since we are querying for it, we probably should add an `INDEX` for the fake_scid.